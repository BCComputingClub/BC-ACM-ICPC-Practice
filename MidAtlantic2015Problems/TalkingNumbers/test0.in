1
222143
1000001
-1
