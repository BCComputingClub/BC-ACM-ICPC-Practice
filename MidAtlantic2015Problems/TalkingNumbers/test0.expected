one
two hundred and twenty-two thousand, one hundred and forty-three
one million and one
